go/docker-compose.yml

```diff
@@ -31,7 +31,7 @@ services:
     volumes:
       - mysql_data:/var/lib/mysql
     ports:
-      - 3306
+      - 3306:3306
```

Comment:
please remove it as soon as you finish your dev

Reply:
there is a `make mysql.shell` helper btw

Reply:
this is for forestadmin to connect via npm start

Reply:
create a complementary docker-compose.yml file that will override this rule if you want, but default docker-compose should expose as less ports as possible

(anyone with a running mysql cannot run this docker-compose file, which is too common)

Reply:
Okay looks already good in docker-compose.dev.yml, will revert it